src/conf_parse.erl

```diff
@@ -20,7 +20,6 @@
 %% conf_parse: for all your .conf parsing needs.
 %%
 %% Copyright (c) 2013 Basho Technologies, Inc.  All Rights Reserved.
-%% Copyright (c) 2019 Pivotal Software, Inc.  All rights reserved.
```

Comment:
just noticed this, i guess this is being manually added to the generated parser, do we want to keep it? @lukebakken